applications/monitoring/values.yaml

```diff
@@ -1,21 +1,119 @@
+# Configuration of Influx endpoint to receive monitoring data.
+config:
+  influxdbHostname: "monitoring.lsst.codes"
+  influxdbOrg: "square"
+
+  # -- Use prometheus config to specify all Prometheus endpoints on services
+  # Except monitoring.  Monitoring is enabled if you're installing this chart,
+  # but only if the influxdb2 component is installed, which is a global
+  # singleton, would you have any Prometheus endpoint to scrape.
+  prometheus:
```

Comment:
Much too long of a default to put into the README, so needs a `@default - See values.yaml` comment.